src/ctapipe/utils/astro.py

```diff
@@ -4,63 +4,194 @@
 not provided by external packages and/or to satisfy particular needs of
 usage within ctapipe.
 """
-from astropy import units as u
+
+import logging
+from copy import deepcopy
+from enum import Enum
+
+import astropy.units as u
 from astropy.coordinates import Angle, SkyCoord
+from astropy.table import Table
+from astropy.time import Time
 
-__all__ = ["get_bright_stars"]
+log = logging.getLogger("main")
 
+__all__ = ["get_star_catalog", "get_bright_stars"]
 
-def get_bright_stars(pointing=None, radius=None, magnitude_cut=None):
-    """
-    Get an astropy table of bright stars.
 
-    This function is using the Yale bright star catalog, available through ctapipe
-    data downloads.
+class StarCatalogues(Enum):
+    Yale = {
+        "directory": "V/50/catalog",
+        "coordinates": {
+            "frame": "icrs",
+            "epoch": "J2000.0",
+            "RA": {"column": "RAJ2000", "unit": "hourangle"},
+            "DE": {"column": "DEJ2000", "unit": "deg"},
+            "pmRA": {"column": "pmRA", "unit": "arcsec/yr"},
```

Comment:
again, units here should not be needed. and the columns are the same in both catalogs.

Reply:
yeah, I didn't delete them for consistency with RA/DE, but of course this can be done along with your comment below on `qunatity` conversion. Do you know if this holds for the majority of the catalogs? I don't have much experience with them...

Reply:
I don't know. But I can only repeat myself, the goal here should not be to define a high-level interface to all catalogs in vizier via astroquery